plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -3439,6 +3438,25 @@ public void testIfDeletesReturnsNumberOfRemovedRows()
         assertUpdate("DELETE FROM " + tableName + " WHERE key = 'two'", 2);
     }
 
+    @Test
```

Comment:
Can we have a test for showcasing that setting an arbitrary property value is not being supported?

```
"ALTER TABLE " + tableName + " SET PROPERTIES foo = 'bar'"
```